cli/js/lib.deno.shared_globals.d.ts

```diff
@@ -1080,6 +1080,7 @@ declare class Worker extends EventTarget {
     options?: {
       type?: "classic" | "module";
       name?: string;
+      deno?: boolean;
     }
```

Comment:
Maybe you could add some TSDoc here explaining how to use this? Ideally with examples.

Reply:
PTAL